Content.Server/Body/Systems/RespiratorSystem.cs

```diff
@@ -372,6 +373,22 @@ private void StopSuffocation(Entity<RespiratorComponent> ent)
 
         _damageableSys.TryChangeDamage(ent, ent.Comp.DamageRecovery);
 
+        var ev = new StopSuffocatingEvent();
+        RaiseLocalEvent(ent, ref ev);
+    }
+
+    private void OnSuffocation(Entity<BodyComponent> ent, ref SuffocationEvent args)
+    {
+        // TODO: This is not going work with multiple different lungs, if that ever becomes a possibility
+        var organs = _bodySystem.GetBodyOrganEntityComps<LungComponent>((ent, null));
+        foreach (var entity in organs)
+        {
+            _alertsSystem.ShowAlert(ent, entity.Comp1.Alert);
+        }
+    }
+
+    private void OnStopSuffocating(Entity<BodyComponent> ent, ref StopSuffocatingEvent args)
+    {
```

Comment:
would it be reasonable to move the body subs outside of respirator system? It'd be more standard to handle it inside BodySystem.